extensions/blocks/membership-button/edit.jsx

```diff
@@ -267,6 +267,31 @@ class MembershipsButtonEdit extends Component {
 		</div>
 	);
 
+	renderDisclaimer = () => {
+		return (
+			<div className="membership-button__disclaimer">
+				{ __( 'Read more about ', 'jetpack' ) }
+				<a
```

Comment:
You'll wanna use `ExternalLink` component here to indicate we're about to open a new tab and leave for external site:

https://github.com/WordPress/gutenberg/tree/d88977b1eb28ba23e760567041ee61f52eca2c7c/packages/components/src/external-link

Reply:
Done!